File: Js/ejer26.js
tipo = 2
compra = 200

if (tipo == 1){
    descuento= compra * 0.20
    total = compra - descuento
    console.log("Cliente VIP")
    console.log("Descuento:", descuento)
    console.log("Total a pagar:", total)
}
else if (tipo == 2) {
    descuento= compra * 0.05
    total = compra - descuento
    console.log("Cliente Normal")
    console.log("Descuento:", descuento)
    console.log("Total a pagar:", total)
}
<<<<<<< HEAD
else;
=======
else
>>>>>>> 1e7f8222b70419d9fdd9b5bfc03ad86b4ef62bbf
    console.log ("Tipo de cliente no valido")

File: python/ejer26.py
tipo = int(input("Ingresa tipo de cliente (1 = VIP, 2 = Normal):"))
compra = int(input("Ingrese el valor de la compra:"))
if tipo == 1:
    descuento = compra * 0.20
    total = compra - descuento
    print("Cliente VIP")
    print("Descuento:", descuento)
    print("Total a pagar:", total)
elif tipo == 2:
    descuento = compra *0.05
    total = compra - descuento
    print("Cliente Normal")
    print("Descuento:", descuento)
    print("Total a pagar:", total)
else:
    print("Tipo de cliente no válido")    